src/devices/Bmp280/Bmp280.sln

```diff
@@ -0,0 +1,36 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
We are not checking in solutions for every binding just to avoid having too many solutions in the project that will slow down nuget restore.